File: chap01/max3.c
#include <stdio.h>

int main(void)
{
 int a, b, c;
 int max;
 printf("Get the maximum value among three variables.\n");
 printf("a : "); scanf("%d", &a);
 printf("b : "); scanf("%d", &b);
 printf("c : "); scanf("%d", &c);
 max = a;
 if(b > max) max = b;
 if(c > max) max = c;

 printf("The maximum values is %d.\n", max);

 return 0;
}

File: chap01/max3.py
max = 0
print(f"Get the maximum value among three variables.", end="\n")
a = int(input("a : ").strip())
b = int(input("b : ").strip())
c = int(input("c : ").strip())
max = a
if(b > max): max = b
if(c > max): max = c
print(f"The maximum value is {max}", end="\n")
